services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,11 @@
 ---
 title: "eth_call"
 ---
+import ParserOpenRPC from "@site/src/components/ParserOpenRPC"
```

Comment:
Clearing out values didn't clear out all param property names in "Customize request"
<img width="464" alt="image" src="https://github.com/user-attachments/assets/9276a1aa-55d9-4888-993f-a6dfb4f86a46">

Reply:
Passing null is not the same as not passing the param: curl https://linea-mainnet.infura.io/v3/<YOUR-API-KEY> \
  -X POST \
  -H "Content-Type: application/json" \
  -d '{
    "jsonrpc": "2.0",
    "method": "eth_call",
    "params": [{"gasPrice":"0x0","accessList":[],"to":null,"input":"0x18160ddd"},"0x980a20"],
    "id": 1
  }'